test/arrays.js

```diff
@@ -194,6 +194,62 @@ describe('Array Diff', function() {
     ];
     assert.deepEqual(diff(parent, theirs, mine), expected);
   });
+
```

Comment:
I'm not concerned right now, but definite idea for the future.  I'll create an issue for it.